client/src/pages/DashboardHomePage/DashboardHomePage.js

```diff
@@ -1,7 +1,11 @@
 import React, { Component } from 'react';
```

Comment:
Can you revert this file to the development version since this PR should only include the dropdown changes?